components/editor/CodeBlock.tsx

```diff
@@ -268,16 +268,18 @@ export default function CodeBlock({
             style={theme}
             showLineNumbers={showLineNumbers}
             startingLineNumber={startingLineNumber}
-            lineNumberContainerProps={{
-              className: 'pl-2 float-left left-0 sticky bg-code-editor-dark',
-              style: {}
+            lineNumberContainerStyle={{
+              paddingLeft: '0.5em',
+              background: '#252f3f',
             }}
-            lineNumberProps={(lineNumber: number) => {
+            lineNumberStyle={(lineNumber: number) => {
               const isHighlighted = highlightedLines && highlightedLines.includes(lineNumber);
 
               return {
-                className: `${isHighlighted ? 'bg-code-editor-dark-highlight text-gray-500' : 'text-gray-600'}
-                block pl-2 pr-2`
+                display: 'inline-block',
+                padding: '0 0.5em',
+                background: isHighlighted ? '#252f3f' : 'inherit',
+                color: isHighlighted ? '#A3ACAD' : '#8B9394'
```

Comment:
how you got these colors and properties? is this part of master branch?

Reply:
From https://github.com/asyncapi/website/blob/48cb678afdc9e83424a1a148e2640178f4ae2d96/tailwind.config.js#L122-L125

Reply:
So, why it isn't taking color directly from tailwind?